dxcherry/.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+    "files.exclude": {
+        "out": false // set this to true to hide the "out" folder with the compiled JS files
```

Comment:
Can we get rid of this comments? Maybe we can add `_comment` property to json object?

Reply:
decided ignore this problem